### PR TITLE
Print full command to check oozie job status

### PR DESCRIPTION
**Pull request by mik-laj**
Hello,

This is my idea for implementation. It seems simple to me. The most important thing is that it allows you to accomplish the #168 issues. I use this PR and it works well with me.

Thanks

### CODE REVIEW

File: oozie-to-airflow/run-sys-test [hunk 1]
```diff
@@ -470,12 +470,15 @@ function test_composer {
...
+    # Skip prefix
+    # "job: " = 5 characters
+    JOB_ID=$(echo "JOB_ID_LINE" | cut -c 6-)
     set +x
-    echo "You can run the following commands on dataproc cluster to see the job status"
+    echo "You can run the following commands cluster to see the job status"
     echo
-    echo "oozie job -info <JOB_ID>"
-    echo "oozie job -log  <JOB_ID>"
+    echo "${CMDNAME} -S -- oozie job -info ${JOB_ID@Q}"
```
Comment (potiuk) [inline oozie-to-airflow/run-sys-test]: I don't think we need Q -> it's confusing what it does and is not needed to escape JOB_ID which has no spaces.

File: oozie-to-airflow/run-sys-test [hunk 2]
```diff
@@ -470,12 +470,15 @@ function test_composer {
 
 function test_oozie {
     header "Triggers example application on Oozie: ${LOCAL_APP_NAME}"
-    run_command "sudo oozie job --config ${REMOTE_FILESYSTEM_APP_PROPERTIES} -run"
+    JOB_ID_LINE=$(run_command "sudo oozie job --config ${REMOTE_FILESYSTEM_APP_PROPERTIES} -run" | tee /dev/tty | grep '^job: ')
+    # Skip prefix
+    # "job: " = 5 characters
+    JOB_ID=$(echo "JOB_ID_LINE" | cut -c 6-)
```
Comment (potiuk) [inline oozie-to-airflow/run-sys-test]: JOB_ID_LINE misses $

Comment (mik-laj) [reply oozie-to-airflow/run-sys-test]: Yes. You've right. it looks like I have not sent the latest version to the remote server.

**Comment by mik-laj** [comment]
@potiuk Travis is green. I also commented on all suggestions. PTAL

File: oozie-to-airflow/run-sys-test (hunk 2 above)
Comment (mik-laj) [reply oozie-to-airflow/run-sys-test]: I updated the PR.

File: oozie-to-airflow/run-sys-test (hunk 1 above)
Comment (mik-laj) [reply oozie-to-airflow/run-sys-test]: I updated the PR.

**Comment by potiuk** [comment]
Can you please rebase

**Pull request merged**